README.md

```diff
@@ -8,52 +8,60 @@
 
 An example implementation of the [NLP Sandbox Date Annotator].
 
-## Specification
+## Overview
 
-- Implements the [NLP Sandbox Date Annotator OpenAPI specification]
-- Annotates date strings in clinical notes using simple regular expressions
+This repository provides a Python-Flask implementation of the [NLP Sandbox Date
+Annotator]. The Date Annotator is one of the first NLP Tools that can be
+benchmarked on nlpsandbox.io. A Date Annotator takes as input one clinical notes
+as well as information about the patient and outputs a list of predicted date
+annotations found in the clinical note.
 
-NLP Sandbox API implemented:
+- Date Annotator API version: 0.3.1
+- Tool version: 0.3.2
+- Docker image: [nlpsandbox/date-annotator-example]
 
-| API | Type | Version |
-|---|---|---|
-| [Date Annotator](/openapi/date-annotator) | `nlpsandbox:date-annotator` | 0.3.2 |
+## Model
 
-## Usage
+This NLP tool relies on simple regular expressions to identify the location of
+date strings in a clinical note.
+
+This implementation is provided as an example that Developers can use to quick
+start the development of a new model by creating a repository from this [GitHub
+template] (see below). The Docker image built and published automatically by the
```

Comment:
This sentence is a bit run-on. Maybe something like this instead?

"The CI/CD workflow of this repository will automatically build and publish a Docker image to DockerHub.  The model can then be submitted as-is, if you wish to benchmark its performance -- just don't expect a high performance!"

Reply:
Done